apiserver/common/modelmanagerinterface.go

```diff
@@ -151,6 +151,7 @@ func (st modelManagerStateShim) GetBackend(modelUUID string) (ModelManagerBacken
 	}
 	otherModel, err := otherState.Model()
```

Comment:
This looks like it may be a problem is a few places. Did you check the entire codebase in case there are other places?